core/matchers/src/main/kotlin/au/com/dius/pact/core/matchers/MatcherExecutor.kt

```diff
@@ -119,10 +105,51 @@ fun <M : Mismatch> domatch(
             matchMaxType(matcher.max, path, expected, actual, mismatchFn)
     is IncludeMatcher -> matchInclude(matcher.value, path, expected, actual, mismatchFn)
     is NullMatcher -> matchNull(path, actual, mismatchFn)
+    is IgnoreOrderMatcher -> matchIgnoringOrder(path, expected, actual, mismatchFn)
+    is MinEqualsIgnoreOrderMatcher -> matchMinEquals(matcher.min, path, expected, actual, mismatchFn)
+    is MaxEqualsIgnoreOrderMatcher -> matchMaxEquals(matcher.max, path, expected, actual, mismatchFn)
+    is MinMaxEqualsIgnoreOrderMatcher -> matchMinEquals(matcher.min, path, expected, actual, mismatchFn) +
+            matchMaxEquals(matcher.max, path, expected, actual, mismatchFn)
```

Comment:
wouldn't you want/need to try the more specific cases after the less specific case?

```suggestion
    is MinMaxEqualsIgnoreOrderMatcher -> matchMinEquals(matcher.min, path, expected, actual, mismatchFn) +
            matchMaxEquals(matcher.max, path, expected, actual, mismatchFn)
        is MinEqualsIgnoreOrderMatcher -> matchMinEquals(matcher.min, path, expected, actual, mismatchFn)
    is MaxEqualsIgnoreOrderMatcher -> matchMaxEquals(matcher.max, path, expected, actual, mismatchFn)
    is IgnoreOrderMatcher -> matchIgnoringOrder(path, expected, actual, mismatchFn)
```

Reply:
I can change this.  I was just following the form used for MinType, ...

Reply:
Looking again, the order shouldn't matter...   I'm not sure why I was thinking that did.

Reply:
No, the order doesn't matter.  I thought you wanted the change for aesthetics.